src/Workspaces/Core/Portable/Execution/PinnedRemotableDataScope.cs

```diff
@@ -33,6 +33,11 @@ internal sealed class PinnedRemotableDataScope : IDisposable
             _storages.RegisterSnapshot(this, storage);
         }
 
+        /// <summary>
+        /// This indicates whether this scope is for primary branch or not (not forked solution)
+        /// </summary>
```

Comment:
comments shoudl explain why this is important.  What will consumers use this for?